docs/source/en/_toctree.yml

```diff
@@ -1,144 +1,158 @@
-    - local: tasks/sequence_classification
-      title: Text classification
-    - local: tasks/token_classification
-      title: Token classification
-    - local: tasks/question_answering
-      title: Question answering
-    - local: tasks/language_modeling
-      title: Causal language modeling
-    - local: tasks/masked_language_modeling
-      title: Masked language modeling
-    - local: tasks/translation
-      title: Translation
-    - local: tasks/summarization
-      title: Summarization
-    - local: tasks/multiple_choice
-      title: Multiple choice
-    title: Natural Language Processing
-  - isExpanded: false
+    - local: autoclass_tutorial
+      title: AutoClass API
+    - local: create_a_model
+      title: Use model-specific APIs
+    - local: custom_models
+      title: Share a custom model
+    - local: model_sharing
+      title: Share your model
+    - local: add_new_model
+      title: How to add a model to 🤗 Transformers?
+    - local: task_summary
+      title: What 🤗 Transformers can do
+    - local: tasks_explained
+      title: How 🤗 Transformers solve tasks
+    - local: model_summary
+      title: The Transformer model family
+    - local: attention
+      title: Attention mechanisms
+    - local: bertology
+      title: BERTology
+  - title: Tokenizers
+    isExpanded: false
     sections:
-    - local: tasks/audio_classification
-      title: Audio classification
-    - local: tasks/asr
-      title: Automatic speech recognition
-    title: Audio
-  - isExpanded: false
+    - local: preprocessing
+      title: Preprocess data
+    - local: fast_tokenizers
+      title: Use fast tokenizers from 🤗 Tokenizers
+    - local: tokenizer_summary
+      title: Summary of the tokenizers
+    - local: pad_truncation
+      title: Padding and truncation
+  - title: Deploy to production
+    isExpanded: false
     sections:
-    - local: tasks/image_classification
-      title: Image classification
-    - local: tasks/semantic_segmentation
-      title: Image segmentation
-    - local: tasks/video_classification
-      title: Video classification
-    - local: tasks/object_detection
-      title: Object detection
-    - local: tasks/zero_shot_object_detection
-      title: Zero-shot object detection
-    - local: tasks/zero_shot_image_classification
-      title: Zero-shot image classification
-    - local: tasks/monocular_depth_estimation
-      title: Depth estimation
-    - local: tasks/image_to_image
-      title: Image-to-Image
-    - local: tasks/image_feature_extraction
-      title: Image Feature Extraction
-    - local: tasks/mask_generation
-      title: Mask Generation
-    - local: tasks/knowledge_distillation_for_image_classification
-      title: Knowledge Distillation for Computer Vision
-    title: Computer Vision
-  - isExpanded: false
+    - local: serialization
+      title: Export to ONNX
+    - local: tflite
+      title: Export to TFLite
+    - local: torchscript
+      title: Export to TorchScript
```

Comment:
I think this would be better in a top level instead of under `title: Pretrained models`